FWCore/Services/src/SiteLocalConfigService.cc

```diff
@@ -26,9 +26,36 @@ using namespace xercesc;
 //<<<<<< MEMBER FUNCTION DEFINITIONS                                    >>>>>>
 
 namespace {
+
+  template< class CharType >
```

Comment:
I am still extremely oposed to putting files that have nothing to do with concurrency into FWCore/Concurrency. I'd rather move the Xerces related functions out of FWCore/Concurrency and move them, along with this header only code, into a Utilities/Xerces package.